model/ftn/w3psmcmd.ftn

```diff
@@ -600,48 +602,50 @@
            IF( L > 0 ) THEN
 !! Add sub-grid blocking for refined cells.   JGLi18Apr2018 
               IF( (CTRNX(M)+CTRNX(L)) .GE. 1.96 )  THEN
-!/OMPG!$OMP ATOMIC 
+!/OMPG/!$OMP ATOMIC 
                 FCNt(L) = FCNt(L) - FUTRN
               ELSE IF( ULCFLX(i) .GE. 0.0 ) THEN
-!/OMPG!$OMP ATOMIC 
+!/OMPG/!$OMP ATOMIC 
                 FCNt(L) = FCNt(L) - FUTRN*CTRNX(L) 
               ELSE
-!/OMPG!$OMP ATOMIC 
+!/OMPG/!$OMP ATOMIC 
                 FCNt(L) = FCNt(L) - FUTRN*CTRNX(L)*CTRNX(M) 
               ENDIF 
-!/OMPG!$OMP ATOMIC 
-              AFCN(L) = AFCN(L) - FUMD(i)*UCFL(L)*FMR + FUDIFX(i)
+!/OMPG/!$OMP ATOMIC 
+              ! ChrisB: Re-arranged the RHS term below to make it
+              ! valid for OMP ATMOIC directive.
+              AFCN(L) = AFCN(L) - (FUMD(i)*UCFL(L)*FMR - FUDIFX(i))
            ENDIF
            IF( M > 0 ) THEN
 !! Add sub-grid blocking for refined cells.   JGLi18Apr2018 
               IF( (CTRNX(M)+CTRNX(L)) .GE. 1.96 )  THEN
-!/OMPG!$OMP ATOMIC 
+!/OMPG/!$OMP ATOMIC 
                 FCNt(M) = FCNt(M) + FUTRN
               ELSE IF( ULCFLX(i) .GE. 0.0 ) THEN
-!/OMPG!$OMP ATOMIC 
+!/OMPG/!$OMP ATOMIC 
                 FCNt(M) = FCNt(M) + FUTRN*CTRNX(M)*CTRNX(L) 
               ELSE
-!/OMPG!$OMP ATOMIC 
+!/OMPG/!$OMP ATOMIC 
                 FCNt(M) = FCNt(M) + FUTRN*CTRNX(M) 
               ENDIF 
-!/OMPG!$OMP ATOMIC 
+!/OMPG/!$OMP ATOMIC 
               AFCN(M) = AFCN(M) + FUMD(i)*UCFL(M)*FMR - FUDIFX(i)
```

Comment:
Change _x operator expr_  to _x operator (expr)_: AFCN(M) = AFCN(M) + ( FUMD(i)*UCFL(M)*FMR - FUDIFX(i) )